cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/BudgetDTO.java

```diff
@@ -18,104 +15,94 @@
 @ApiModel( description = "Budget estimate" )
 public class BudgetDTO extends BaseEntityDTO
 {
-    @ApiModelProperty( value = "Country code", readOnly = true )
+    @ApiModelProperty( "Country code" )
     @NotNull
     private String nation = "IT";
 
-    @ApiModelProperty( value = "URL of the chosen cloud provider", readOnly = true )
+    @ApiModelProperty( "URL of the chosen cloud provider" )
     @NotNull
     @Valid
     private UrlEntityDTO cloudProvider;
 
-    @ApiModelProperty( value = "URL of the chosen product version", readOnly = true )
+    @ApiModelProperty( "URL of the chosen product version" )
     @NotNull
     @Valid
     private UrlEntityDTO configuration;
 
-    @ApiModelProperty( value = "Trial duration in days", readOnly = true )
+    @ApiModelProperty( "Trial duration in days" )
     private Integer trialLength;
 
-    @ApiModelProperty( value = "Billable entries for the estimate", readOnly = true )
+    @ApiModelProperty( "Billable entries for the estimate" )
     private List<OrderLineDTO> lines = new LinkedList<>();
 
-    @ApiModelProperty( value = "Lines billed at the end of the billing period", readOnly = true )
+    @ApiModelProperty( "Lines billed at the end of the billing period" )
     private List<OrderLineDTO> postLines;
 
-    @ApiModelProperty( value = "Lines directly billed to the customer", readOnly = true )
+    @ApiModelProperty( "Lines directly billed to the customer" )
     private List<OrderLineDTO> upsoldLines;
 
-    @ApiModelProperty( value = "If this invoice is subject to the Italian electronic document tax", readOnly = true )
+    @ApiModelProperty( "If this invoice is subject to the Italian electronic document tax" )
     private boolean stampDuty = false;
 
-    @ApiModelProperty( value = "Currency of the prices for the estimate", readOnly = true )
+    @ApiModelProperty( "Currency of the prices for the estimate" )
     private String currency;
 
-    @ApiModelProperty( value = "Value added tax, in percentage", readOnly = true )
+    @ApiModelProperty( "Value added tax, in percentage" )
     @JsonInclude ( JsonInclude.Include.NON_EMPTY )
     private BigDecimal VAT;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "VAT" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS00116)